metadata/definition/definition.go

```diff
@@ -34,6 +41,39 @@ type ServiceDefinition struct {
 	Types         []TypeDefinition
 }
 
+func (def ServiceDefinition) ToBytes() ([]byte, error) {
+	return json.Marshal(def)
+
+}
+
+func (def ServiceDefinition) String() string {
+	var methodStr string
+	for _, m := range def.Methods {
+		var paramType string
+		for _, p := range m.ParameterTypes {
+			paramType = paramType + fmt.Sprintf("{type:%v}", p)
+		}
+		var param string
+		for _, d := range m.Parameters {
+			param = param + fmt.Sprintf("{id:%v,type:%v,builderName:%v}", d.Id, d.Type, d.TypeBuilderName)
+		}
+		methodStr = methodStr + fmt.Sprintf("{name:%v,parameterTypes:[%v],returnType:%v,params:[%v] }", m.Name, paramType, m.ReturnType, param)
```

Comment:
Use `bytes.Buffer` or `string.Builder` to generate `paraType`, `param` and `methodStr`.

Reply:
nice idea.